cmake/globalmacros.cmake

```diff
@@ -259,7 +262,13 @@ function(ivw_register_modules retval)
                 list(APPEND modules ${mod})
                 ivw_private_get_ivw_module_name(${module_path}/${dir}/CMakeLists.txt name)
                 ivw_private_get_ivw_module_version(${module_path}/${dir}/CMakeLists.txt version)
-                ivw_private_setup_module_data(NAME ${name} VERSION ${version} DIR ${dir} BASE ${module_path})
+                ivw_private_setup_module_data(
+                    NAME ${name} 
+                    GROUP "inviwo-${group_name}-modules" 
```

Comment:
Can't the group be just the name, do we need to prepend and append inviwo- -modules? 
I think that it would look much cleaner without it

Reply:
the prefix makes them sort together which I like. the postfix was mostly because I found just having "inviwo-core" a strange since we already have that as a different target

Reply:
an option ( which I know peter doesn't favour) is to have them in a folder, i.e modules/inviwo-core modules/inviwo-misc etc. Then the default group name could be other, or nothing (so they appear in modules/others or directly in modules). 
Then we could also drop the inviwo prefix

Reply:
my issue with that is that it will make it even harder to find "your" code. I also use the collapse all in vs often, and then having more levels means more clicks to get to where i want to go

Reply:
I have had the folder solution locally for quite some time, and use the collapse all feature quite a lot and haven't found it annoying :) And it is only one more click ;)